pybossa/api/__init__.py

```diff
@@ -154,8 +154,10 @@ def _retrieve_new_task(project_id):
 @crossdomain(origin='*', headers=cors_headers)
 def get_km_square(project_short_name=None):
     if project_short_name:
+        project = project_repo.get_by_shortname(project_short_name)
```

Comment:
@Vullkan92 Here it uses project.id:
https://github.com/AltClick/pybossa-amnesty-microtasking/blob/master/pybossa/repositories/task_repository.py#L140

1) Where else does it use any other property of the Project object?
2) What is stopping you from creating a Project object that just has an id property, like so: project = { "id": the_id }?

3) Even so, what is stopping you from creating a new function for your need, as I explained earlier: _task_repo.get_tasks_redundancy_by_project_name function(project_id)_? 

4) Why not create that function and then refactor the _task_repo.get_tasks_redundancy_ function to remove duplicate code and just have it call _task_repo.get_tasks_redundancy_by_project_name function(project.id)_ instead?

Finally, setting aside all of the above. There is a way to achieve this without having to query the sql database at all. Remember that you have the javascript task object in the task presenter.

5) Does the javascript task object in the task presenter have the redundancy number as a property? If so, what is it and how can you use it?

Please answer all the questions I asked. I need to make sure you understand everything.